workflows/edit_eml/set_parties.Rmd

```diff
@@ -20,7 +20,7 @@ JC_creator <- arcticdatautils::eml_creator(given_names = "Jeanette",
                                            phone = "123-456-7890", 
                                            userId = "https://orcid.org/WWWW-XXXX-YYYY-ZZZZ",
                                            address = NCEASadd)
-eml@dataset@creator <- c(JC_creator)
+doc$dataset$creator <- JC_creator
```

Comment:
Apparently you can't comment on unchanged lines but should the `JC_creator` variable be created with `eml$creator`?

Reply:
this uses the arcticdatautils helper `eml_creator` instead